packages/@ourworldindata/components/src/styles/typography.scss

```diff
@@ -360,7 +360,7 @@ body {
 @mixin body-3-regular {
     font-family: $sans-serif-font-stack;
     font-size: 0.875rem;
-    line-height: 1.5;
+    line-height: 1.125rem;
```

Comment:
Just checking: this (and below) changes how the text looks in many places. Was the change intentional?

Reply:
Yup, I'm not sure if this was a transcription error initially or if the typography's been updated since we first committed these, but in Figma this is what the line heights are now.